bit9PlatformAPI/examples/connectors/Metascan.py

```diff
@@ -0,0 +1,247 @@
+"""
+First off, I'm not a programmer so any issues that you have in running this script in your environment are not my fault!
```

Comment:
Totally understood :) let's take this disclaimer out of here and leave it to the "official" disclaimer.